pkgs/development/libraries/libhugetlbfs/default.nix

```diff
@@ -40,10 +40,14 @@ stdenv.mkDerivation rec {
   installTargets = [ "install" "install-docs" ];
 
   meta = with lib; {
-    broken = (stdenv.isLinux && stdenv.isAarch64);
     description = "library and utilities for Linux hugepages";
     maintainers = with maintainers; [ qyliss ];
     license = licenses.lgpl21Plus;
     platforms = platforms.linux;
+    badPlatforms = [
+      systems.inspect.platformPatterns.isStatic
+      systems.inspect.patterns.isMusl
+      systems.inspect.patterns.isAarch64
```

Comment:
Does libhugetlbfs not work on aarch64?
The project does seem to support it.

Reply:
At least currently it doesn't compile, and it was marked broken on that platform before.

Reply:
It compiles for me